scripts/install-windows-prereqs.ps1

```diff
@@ -588,28 +589,82 @@ function Install-Docker {
     Write-Output '[WARNING] Docker is no longer a prerequisite and installation thereof is deprecated.'
 }
 
-try {
+function Validate-SkipTasks {
+    Param(
+        [Parameter(Mandatory=$true)][array]$SkipTaskLower
+    )
+    $ValidSet = @("Nuget3", "Nuget6", "Python", "VisualStudio", "LLVM", "Git", "Shellcheck", "NSIS", "Docker", "OpenSSL", "VCRuntime")
+    foreach($item in $SkipTaskLower) {
+        if($ValidSet -notcontains $item) {
+            Write-Output "$item is not a valid package to skip."
+            return
+        }
+        Write-Output "Skipping $PackageToInstall"
```

Comment:
Suggestion 1: since you would print two "skipping" messages per skipped package, you could have cleaner code by keep this line and instead remove the messages that you print out as you install each package.

Suggestion 2: consider using [Write-Information](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/write-information?view=powershell-5.1) here